tests/multiple_gtfs_rts.rs

```diff
@@ -0,0 +1,218 @@
+use actix_web::http;
+use actix_web::HttpMessage;
+use prost::Message;
+use transpo_rt::context::{DatasetInfo, Datasets};
+use transpo_rt::siri_model::{DateTime, SiriResponse};
+use transpo_rt::transit_realtime;
+mod utils;
+
+fn string(time: &Option<DateTime>) -> Option<String> {
+    time.as_ref().map(|t| t.to_string())
+}
+
+pub fn make_test_server() -> actix_web::test::TestServer {
+    utils::init_logger();
+    let period = transpo_rt::context::Period {
+        begin: chrono::NaiveDate::from_ymd(2018, 12, 15),
+        horizon: chrono::Duration::days(1),
+    };
+
+    let make_server = move || {
+        let dataset_infos = Datasets {
+            datasets: vec![DatasetInfo::new_default(
+                "fixtures/gtfs.zip",
+                &[
+                    mockito::SERVER_URL.to_string() + "/gtfs_rt_1",
+                    mockito::SERVER_URL.to_string() + "/gtfs_rt_2",
+                ],
+            )],
+        };
+        let dataset_actors_addr = transpo_rt::server::create_all_actors(&dataset_infos, &period);
+        transpo_rt::server::create_server(&dataset_actors_addr, &dataset_infos)
+    };
+
+    actix_web::test::TestServer::with_factory(make_server)
+}
+
+pub fn run_gtfs_rt_server(
```

Comment:
Can't you use the one of utils?

Reply:
it's not the same code, but you're right it can be factorized